core/domain/exp_jobs_one_off.py

```diff
@@ -470,3 +471,115 @@ def map(item):
     @staticmethod
     def reduce(key, values):
         yield (key, values)
+
+
+class ExplorationContentValidationJob(jobs.BaseMapReduceOneOffJobManager):
+    """Job that checks the html content of an exploration and validates it.
+    """
+
+    @classmethod
+    def entity_classes_to_map_over(cls):
+        return [exp_models.ExplorationModel]
+
+    @staticmethod
+    def map(item):
+
+        def find(key, dictionary):
+            """Finds the value of the key inside all the nested dictionaries
+            in a given dictionary.
+
+            Args:
+                key: str. The key whose value is to be found.
+                dictionary: dict or list. The dictionary or list in which the
+                    key is to be searched.
+
+            Returns:
+                list. The values of the key in the given dictionary.
+            """
+            if isinstance(dictionary, list):
+                for d in dictionary:
+                    if isinstance(d, (dict, list)):
+                        for result in find(key, d):
+                            yield result
+            else:
+                for k, v in dictionary.iteritems():
+                    if k == key:
+                        yield v
+                    elif isinstance(v, dict):
+                        for result in find(key, v):
+                            yield result
+                    elif isinstance(v, list):
+                        for d in v:
+                            if isinstance(d, (dict, list)):
+                                for result in find(key, d):
+                                    yield result
+
+        if item.deleted:
+            return
+
+        exploration = exp_services.get_exploration_from_model(item)
+
+        allowed_parent_list = {
+            'p'   : ['blockquote', 'div', 'pre', '[document]'],
```

Comment:
Hi @bansalnitish! We do not prefer whitespace(s) after `:`. The lint checks fail due to this reason. Also, can you please sync your branch with develop? Some new lint rules have been added recently. Thanks!

Reply:
Hi @apb7! Thanks for the review. I think whitespace(s) are not preferred before `:`. There were errors which were resolved when I removed spaces before the colon.
Thanks!

Reply:
Oh. My bad @bansalnitish! Thanks :)

Reply:
No problem :)